src/garage/torch/distributions/tanh_normal_distribution.py

```diff
@@ -0,0 +1,88 @@
+import numpy as np
```

Comment:
Wait, so are the contents of this file actually used? I guess they're correct anyways, so we might reasonably want to include them.